ee/codegen/src/generators/nodes/generic-node.ts

```diff
@@ -119,8 +119,19 @@ export class GenericNode extends BaseSingleFileNode<
           ) {
             const rawBlocks = blocks.value.value as PromptBlockSerializer.Raw[];
             const deserializedBlocks: PromptBlockType[] = rawBlocks.map(
-              // @ts-ignore
-              (block) => PromptBlockSerializer.parse(block).value
+              (block) => {
+                const parseResult = PromptBlockSerializer.parse(block);
+                if (parseResult.ok) {
+                  // TODO: Remove `as` once other types of blocks are supported
+                  return parseResult.value as PromptBlockType;
+                } else {
+                  throw new Error(
```

Comment:
[nit] Wherever possible, we should throw some extension of BaseCodegenError and catch them with workflow context so we don't kill the full code generation process